src/target.ts

```diff
@@ -105,9 +107,7 @@ export class Target {
   }
 
   private getFilename(extension) {
-    // TODO: use something unique instead of measurement in filename
-    // as measurement field exists only in influxDB metric
-    return `${this.metric.targets[0].measurement}.${this.from}-${this.to}.${extension}`;
+    return `${this.datasourceName}.${this.from}-${this.to}.${extension}`;
```

Comment:
if we create two exports for different metrics but for same datasource, will we rewrite old export file for same time range?

Reply:
Unfortunately
Didn't come up with the metric-unique field yet
I would create issue about that, thx

Reply:
may be it would be better to use datasourceName.current epoch time.extension ?

Reply:
good idea 👍